### PR TITLE
fix: update viper mod

### PR DESCRIPTION
**Pull request by gris**

### CODE REVIEW

**Review by haaawk — APPROVED** [review]

File: internal/settings/settings.go [hunk 1]
```diff
@@ -144,6 +144,7 @@ func (s *Settings) SetAutoupdate(autoupdate string) {
 
 func (s *Settings) SetLastUpdateCheck(t int64) {
 	config := viper.GetStringMap("config")
+	fmt.Println(config)
```
Comment (haaawk) [inline internal/settings/settings.go:147]: This shouldn't be here.

Comment (gris) [reply internal/settings/settings.go:147]: It is just for debugging purposes :)

Comment (gris) [reply internal/settings/settings.go:147]: I mean, I'm just testing here bc I can't reproduce in my local env

Comment (haaawk) [reply internal/settings/settings.go:147]: I figured but we shouldn't merge it, should we?

Comment (gris) [reply internal/settings/settings.go:147]: Nope!

**Pull request closed without merging**